PYTHON_CDM_Conversion_code/daily_to_cdm_header_v1.py

```diff
@@ -1,42 +1,118 @@
+#!/usr/bin/env python3
 # -*- coding: utf-8 -*-
 """
-Created on Wed Nov 24 10:26:06 2021
+Convert daily OBSERVATIONS files to HEADER .psv files (one per station).
+
+
+Call in one of three ways using:
+
+>python daily_to_cdm_header_v1.py --station STATIONID
+>python daily_to_cdm_header_v1.py --subset FILENAME
+>python daily_to_cdm_header_v1.py --run_all
+>python daily_to_cdm_header_v1.py --help
+
+#Created on Thu Nov 11 16:31:58 2021
 
 @author: snoone
-"""
 
+edited: snoone, February 2022
+"""
 
 import os
 import glob
 import pandas as pd
 pd.options.mode.chained_assignment = None  # default='warn'
+import utils
+print(dir(utils))

```

Comment:
This isn't necessary any more (to list the functions and objects accessible from `utils`). Remove this print statement